services/filemanager/digitalOceanSpacesManager.go

```diff
@@ -158,15 +206,39 @@ func GetDOSpacesConfig(config map[string]interface{}) *DOSpacesConfig {
 	if config["accessKey"] != nil {
 		accessKey = config["accessKey"].(string)
 	}
-	return &DOSpacesConfig{Bucket: bucketName, EndPoint: endPoint, Prefix: prefix, AccessKeyID: accessKeyID, AccessKey: accessKey}
+	if config["region"] != nil {
+		tmp := config["region"].(string)
+		region = &tmp
+	}
+	if config["forcePathStyle"] != nil {
+		tmp := config["forcePathStyle"].(bool)
```

Comment:
Given incorrect type: `config["forcePathStyle"] = "foo"`

 this code will panic. Are we ok with this ?